pshtt/utils.py

```diff
@@ -75,3 +75,28 @@ def configure_logging(debug=False):
         log_level = logging.WARNING
 
     logging.basicConfig(format='%(message)s', level=log_level)
+
+
+# Ensure domains given are not prepended with www/https/http/combination of both.
+def format_domain(domain_name):
+    # Check for www. prepended
+    if domain_name.startswith("www."):
+        return domain_name[4:]
+    # Check for https prepended, account for www. as well
+    if domain_name.startswith("https://"):
+        return format_domain(domain_name[8:])
+
+    # Check for http:// prepended, account for www. as well
+    if domain_name.startswith("http://"):
+        return format_domain(domain_name[7:])
```

Comment:
I am curious why we wouldn't want to correct to just the domain names without the prefixes? My interpretation of the fix was to remove any prefixed www or protocol. Is there a functional reason for only removing a single instance of this?